### PR TITLE
adding traffic_signals:sound tag [ready for review]

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes part of #574 

I want to test is it flood of "no" answers. It is also likely that it needs enabling/disabling for specific countries.

### CODE REVIEW

**Comment by matkoniecz** [comment]
Is anybody aware whatever there are countries with traffic lights where sounds for people with poor eyesight are really rare? I plan to run checks based on raw OSM data but this kind of data is biased in many ways.

**Comment by westnordost** [comment]
Hmm, I would rather like to have a whitelist rather than a blacklist, I think. On the other hand, it is really difficult to research this information, i.e. Google Streetview etc. is not helpful here obviously, so it may be a better idea to start off with enabled-almost-everywhere and then exclude countries bit by bit where it turns out that they do not have this at all.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -622,5 +622,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_tracktype_grade3">Mixture</string>
     <string name="quest_tracktype_grade4">Mostly soft</string>
     <string name="quest_tracktype_grade5">Soft</string>
+    <string name="quest_traffic_signals_sound_title">Are there sound signals for visually impaired people at this crossing?</string>
```
[the remark this answers is not in the document]
Comment (matkoniecz) [reply app/src/main/res/values/strings.xml]: "Are there sound signals for blind at this crossing?" is certainly better.

"Are there sound signals for blind here?" I was not immediately sure, but it also looks as an improvement. It is less explicit, but quest icon is explaining well the situation.

[the remark this answers is not in the document]
Comment (matkoniecz) [reply app/src/main/res/values/strings.xml]: I commited changes that follow this suggestions.

**Comment by matkoniecz** [comment]
As expected, checking raw OSM data was not too helpful. There are some obvious cases for inclusion, but most countries are completely missing this tag.

At this point I would recommend rolling it out everywhere and checking statistics in month or two.

But restrictive whitelist and waiting for reports from people who want this quest in their country would be also OK.

**Comment by westnordost** [comment]
The quest is pretty low prio, so that's okay

**Comment by matkoniecz** [comment]
@westnordost BTW, you may want to look at #574

It has three suggestions, this PR implements one. I think that from two remaining ones - first is is not suitable for quest and second is not worth implementing.

If you think that arrow quest is worth implementing - I may do that, and keep it as low priority Germany-only quest.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -624,5 +624,5 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_tracktype_grade5">Soft</string>
     <string name="quest_building_underground_name_title">Is %s completely underground?</string>
     <string name="quest_building_underground_title">Is this building completely underground?</string>
-
+    <string name="quest_traffic_signals_sound_title">Are there sound signals for blind here?</string>
```
Comment (westnordost) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="quest_traffic_signals_sound_title">Are there sound signals for the blind here?</string>
```

**Comment by kjonosm** [comment]
The limited choice of only `yes` and `no` leads to questions, see e.g. [this](https://www.openstreetmap.org/note/3458910) note. As documented in the [OSM wiki](https://wiki.openstreetmap.org/wiki/Key:traffic_signals:sound), could you add additional values `locate` and `walk` as possible answers?

**Comment by matkoniecz** [comment]
1) limited number of notes if fine

2) if changes are needed, please create a new issue

**Comment by mnalis** [comment]
Also, if I understand wiki definition for `locate` and `walk`, and if I understood that autotranslated note; even if those two additional options were present, it would not help to avoid [that note](https://www.openstreetmap.org/note/3458910) (i.e. if there is not change in sound between red and green; it is neither `locate` nor `walk`).

**Pull request merged**